.githooks/run-hooks

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+hook="${0##*/}"
+dotd="${0}.d"
+orig="${0%/*}/../.git/hooks/$hook"
+
+exec 1>&2
+
+for h in "$dotd"/???*; do
+    case "$h" in
+        *~|*.swp)
+            continue
+            ;;
+        [0-9]*-*)
```

Comment:
Maybe we should document somehow (more clearly, somewhere), that the hooks in the drop-in dir have strict naming requirements (otherwise they are silently ignored). WDYT?